string beacon_match_name 
Pose[] points
Pose centroid
uint32 num_points
